openvdb/cmd/openvdb_print.cc

```diff
@@ -11,13 +11,116 @@
 #include <openvdb/openvdb.h>
 #include <openvdb/util/logging.h>
 
+#include <openvdb/points/PointDataGrid.h>
+
+using namespace openvdb;
+using namespace openvdb::points;
```

Comment:
If we're to do this, I'd remove all usage of the `openvdb::` namespace - but as you've opted for auto in most places, I don't think we even need this and can use `openvdb::points` where necessary